packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -98,8 +100,23 @@ export class InsuredBridgeL1Client {
   }
 
   async calculateRealizedLpFeePctForDeposit(deposit: Deposit): Promise<BN> {
```

Comment:
this is the primary logic changed in the client. effectively fetch the liquid reserve ratio and call into the helper method for `calculateRealizedLpFeePct`

Reply:
Do you think we'll want to add some sort of error margin around this value returned? Or add in that margin of error into the `Relayer`?

Reply:
there should be no error margin as this computation is done at an exact block number for the exact relay size. We could add one but the bot really should never got this wrong. wdyt?

Reply:
Ah, agreed then! I think this is worth adding a comment about how its deterministic what the fee should be if you have the correct block number to search for.